Vn-OKTA-396302-Add note about issuer to Node Express Quickstart Guide (#2171)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/go/configmid.md

```bash
CLIENT_ID={yourClientId}
CLIENT_SECRET={yourClientSecret}
ISSUER=https://{yourOktaDomain}.com/oauth2/{authServerId}
```

> If you are using the [default Custom Authorization Server](/docs/concepts/auth-servers/#default-custom-authorization-server), set `{authServerId}=default`. If you are using another [Custom Authorization Server](/docs/concepts/auth-servers/#custom-authorization-server), set `{authServerId}` to the custom Authorization Server ID.

Reviewer: ```suggestion
If you're using the [default Custom Authorization Server](/docs/concepts/auth-servers/#default-custom-authorization-server), set `{authServerId}=default`. If you're using another [Custom Authorization Server](/docs/concepts/auth-servers/#custom-authorization-server), set `{authServerId}` to the custom Authorization Server ID.
```
